Tests/Data/Parabolic/Richards/RichardsFlow_2d_small_adaptive_dt.prj

```diff
@@ -109,6 +109,7 @@
                     <each_steps>100000000</each_steps>
                 </pair>
             </timesteps>
+            <specific_times> 50.0 100.0 500.</specific_times>
```

Comment:
I'm not sure about the "specific" here. Just my opinion, other comments are welcome.
I'd rather call this `fixed_output_times`, because for me, the "specific" alone does not necessary mean "output" in this context.

Reply:
Changed the name to `specified times`.